lib/locale.js

```diff
@@ -3,14 +3,16 @@ import { getLocales } from './storage';
 
 export const allLocales = jsonLocales;
 
+const missingLocaleStr = "(MISSING_LOCALE)";
+
 export default function resolveLocale(locale = 'en', overrideLocales) {
   const returnableFunction = localeObj => (key) => {
     const locales = localeObj[locale];
 
     let localeStr = locales[key];
 
     if (typeof localeStr === 'undefined') {
```

Comment:
Why a separate specific check here rather than adding to and returning from the or chain below?